examples/.clang-format

```diff
@@ -1,6 +1,7 @@
 #
 # The clang-format (Clang 6) style file used by deal.II for the examples.
-# In contrast to the general one, header files are not sorted.
+# In contrast to the general one, header files are not sorted and we do
+# not require return types to be on a separate line.
```

Comment:
Nice!

Reply:
By the way, can these files be hierarchical? E.g., in this file do something like and `#include` of the master style and then simply override the settings we care about?

Reply:
Not that I know of.